src/item/data/item_type.py

```diff
@@ -31,5 +31,6 @@ class ItemType(Enum):
     Wand = "wand"
     TemperManual = "temper manual"
     # Custom Types
+    Incense = "incense"
```

Comment:
Also add to assets and to gen_data

Reply:
I'm not too familiar with what that code is doing but I think I did it